tests/transactions/builder/test_htlc_claim.py

```diff
@@ -1,58 +1,91 @@
+import binascii
+import pytest
 from crypto.configuration.network import set_network
 from crypto.constants import TRANSACTION_HTLC_CLAIM, TRANSACTION_TYPE_GROUP
 from crypto.networks.devnet import Devnet
+from crypto.exceptions import ArkSerializerException
 from crypto.transactions.builder.htlc_claim import HtlcClaim
 
 set_network(Devnet)
 
 
-def test_htlc_claim_transaction():
+def test_htlc_claim_transaction_ok():
     """Test if timelock transaction gets built
     """
     lock_transaction_id = '943c220691e711c39c79d437ce185748a0018940e1a4144293af9d05627d2eb4'
 
-    # This should be the hashed secret used for HTLC Lock transaction
-    unlock_secret = 'my secret that should be 32bytes'
+    # Secret required to unlock and claim funds from an HTLC. This secret should is revealed to you
+    # on the blockchain which you can then use to claim funds.
+    unlock_secret = '6434323233626639336532303235303561366135303134323161383864396661'
 
     transaction = HtlcClaim(lock_transaction_id, unlock_secret)
-    transaction.set_type_group(TRANSACTION_TYPE_GROUP.CORE)
```

Comment:
these ones are not needed as the type group is set directly in `HtlcClaim`'s init class.